eth/contracts/TinyMine.sol

```diff
@@ -90,7 +90,7 @@ contract TinyMine is ITileContract {
     }
 
     modifier closeToMyself(Coords memory selfCoords) {
-        require(connectedWorld.playerInited(msg.sender), "Not an exgrasia player");
+        require(connectedWorld.isPlayerInit(msg.sender), "Not an exgrasia player");
```

Comment:
it looks like these tile contracts are definitely not upgradeable so I think keeping the same name/abi signature in the main contract might be necessary to not break support for already deployed tiles, right?

Reply:
revert the name change :)